src/Components/ClustersListTable/ClustersListTable.js

```diff
@@ -137,27 +148,88 @@ const ClustersListTable = ({
     }
   }, [filters, filterBuilding]);
 
-  const buildFilteredRows = async (items) => {
-    setRowsFiltered(false);
+  const buildFilteredRows = (items) => {
     const filtered = items.filter((it) => {
       return passFiltersCluster(it, filters);
     });
-    const clusterArr = filtered.map((cluster) => cluster.cluster_id);
+
+    const mapped = filtered.map((it) => {
+      return {
+        it,
+        cells: [
+          it.cluster_name || it.cluster_id,
+          it.cluster_version,
+          it.total_hit_count,
+          it.hits_by_total_risk?.[4] || 0,
+          it.hits_by_total_risk?.[3] || 0,
+          it.hits_by_total_risk?.[2] || 0,
+          it.hits_by_total_risk?.[1] || 0,
+          it.last_checked_at,
+        ],
+      };
+    });
```

Comment:
Not completely happy about this, you could say we're doing it once here and later with the labels, but the data needs to be properly sorted, so we don't need to request additional data.